<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseDialog</class>
 <widget class="QDialog" name="ExpenseDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>394</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Add expense</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDialogButtonBox" name="btnBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>165</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbCategory">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="maxVisibleItems">
    <number>20</number>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCategory">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnAddCategory">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>355</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Title / Shop Name</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbExpense">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>155</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <double>1000000000000000042420637374017961984.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lblCost">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Cost: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lblComment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtComment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>355</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbbCurrencyCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblCurrencyCode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Currency Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblExchangeRate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exchange Rate</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnDownloadExchangeRate">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset theme="emblem-downloads">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbExchangeRate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>ExpenseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>ExpenseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
